docs/user/docs/tools/capella/teamforcapella/project-management/project-management.md

```diff
@@ -0,0 +1,54 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# TeamForCapella project management
```

Comment:
I was wondering whether we want to have a look at using [title case](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style) throughout our docs or whether that is maybe just a minor enhancement

Reply:
As this is not consistently used currently - let's address it in another PR. We agreed on using the "Chicago Manual of Style" for all titles, but should also apply to all existing titles.

Reply:
EDIT: Changed the one specific occurrence.